src/modules/common/components/customRoute/index.js

```diff
@@ -37,19 +37,23 @@ const CustomRoute = ({ path, exact, isPrivate, forbiddenTokens, component, t, hi
   if (isPrivate && !isAuthenticated) {
     return (
       <Redirect
-        to={`${routes.login.path}?referrer=${path.replace(/\/(send|vote)/, '')}&${search.replace(
-          /^\?/,
+        to={`${routes.manageAccounts.path}?referrer=${path.replace(
+          /\/(send|vote)/,
           ''
-        )}`}
+        )}&${search.replace(/^\?/, '')}`}
```

Comment:
- When you put `send|vote` you're limiting the correction to those two modals. Same goes to the `search.replace` usage. There's room for improvement.

Reply:
@reyraa Fixed